xbmc/utils/CPUInfo.cpp

```diff
@@ -756,7 +755,6 @@ bool CCPUInfo::readProcStat(unsigned long long& user, unsigned long long& nice,
     }
     catch (...)
```

Comment:
Something in the try block can still throw an exception?

Reply:
no, theoretically.